sourcegraph/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "integration_id": "sourcegraph-datadog-services-map-extension",
+  "categories": [
+    "monitoring",
+    "collaboration",
+    "source control"
+  ],
+  "creates_events": false,
+  "display_name": "Sourcegraph Datadog Services Map Extension",
+  "manifest_version": "2.0.0",
+  "guid": "b647480e-bdfb-4cf6-baa2-e773f69ac466",
+  "is_public": true,
+  "maintainer": "support@sourcegraph.com",
+  "name": "sourcegraph",
+  "public_title": "Sourcegraph Datadog Services Map Extension",
+  "short_description": "View services dependencies as tooltips on your source code",
+  "description": "View services dependencies as tooltips on your source code on Sourcegraph and your code host",
+  "support": "contrib",
+  "supported_os":  [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "display_on_public_website": true,
+  "app_id": "sourcegraph",
+  "type": "check",
+  "assets": {
+    "service_checks": {
+      "metadata_path": "assets/service_checks.json"
+    }
```

Comment:
```suggestion
```

You can also remove this section along with the `assets/service_checks.json` file.